src/StatusQ/Controls/StatusImageCrop.qml

```diff
@@ -0,0 +1,280 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+
+import QtGraphicalEffects 1.1
+
+import StatusQ.Core.Utils 0.1
+
+/*!
+    \qmltype StatusImageCrop
+    \inherits Item
+    \inqmlmodule StatusQ.Controls
+    \since StatusQ.Controls 0.1
+    \brief Draw a crop-window onto an image
```

Comment:
Same than before. It should be nice to add here something like: Inherits \l{https://doc.qt.io/qt-5/qml-qtquick-item.html}{Item}.